app/views/admin/show.html.erb

```diff
@@ -15,22 +15,22 @@
   </ul>
 
   <div id="application_settings">
-    <%= render partial: "show_settings", locals: { banner_message: banner_message, warning_message: warning_message, banner: banner, smtp_settings: smtp_settings, default_url_options: default_url_options, mode: mode, mode_settings: mode_settings, debug_features: debug_features, server_needs_restart: server_needs_restart } %>
+    <%= render 'show_settings', :banner_message => banner_message, :warning_message => warning_message, :banner => banner, :smtp_settings => smtp_settings, :default_url_options => default_url_options, :mode => mode, :mode_settings => mode_settings, :debug_features => debug_features, :server_needs_restart => server_needs_restart %>
```

Comment:
Is there any way to reuse the hash built in the model containing all of the current settings here?